django/db/models/options.py

```diff
@@ -744,16 +737,38 @@ def get_fields(self, include_parents=True, include_hidden=False):
         - include_hidden:  include fields that have a related_name that
                            starts with a "+"
         """
+        if include_parents is False:
+            include_parents = PROXY_PARENTS
         return self._get_fields(include_parents=include_parents, include_hidden=include_hidden)
 
     def _get_fields(self, forward=True, reverse=True, include_parents=True, include_hidden=False,
-                    export_ordered_set=False):
+                    seen_models=None):
+        """
+        Internal helper fuction to return fields of the model. If forward=True then fields defined
+        on this model are returned. If reverse is True, then relations pointing to this model
+        are returned. The include_hidden fields control returning fields with is_hidden attribute
+        set. The include_parents argument toggles if fields from parent models should be included.
+        It has actually three values, True, False and PROXY_PARENTS. When set to PROXY_PARENTS,
+        the call will return all fields defined for the current model or any of its parents in
+        the parent chain to the model's concrete model.
+        """
+        if include_parents not in (True, False, PROXY_PARENTS):
```

Comment:
Are you using PROXY_PARENTS as some type of flag? We already went through the "flags" path throughout the Meta API refactor and ended by removing it because it required imports.
Even though this is an internal function, I would advise to find a better solution because, personally, I don't think it's good practice to have one argument that can be of different types (in this case, boolean or object) as it increases the risk of bugs, and it also can be confusing to other people. But that's just me.